sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_link_async.py

```diff
@@ -173,10 +174,8 @@ async def _incoming_attach(self, frame):
         self.remote_handle = frame[1]  # handle
         self.remote_max_message_size = frame[10]  # max_message_size
         self.offered_capabilities = frame[11]  # offered_capabilities
-        if self.properties:
```

Comment:
So for removing resetting the link properties here, the only thing I see is that the service sends us back a different microsoft.timeout (6500) than we send (0). I don't think that is info we need to send because it works without, just wanted to note it.

Reply:
That could definitely be important - but it's also not part of pyamqp. That custom property would need to be handled in the EH layer is it's something we should be using.

Reply:
We could use that value when creating the retry policy. `CUSTOM_CONDITION_BACKOFF` has the key `com.microsoft:timeout` with a custom value of 2. The service operation timeout is a separate key that can be optionally set on a link property and that influences quickly the service will timeout waiting for an operation to complete.  All of this housed in the SDK